css-color-4/Overview.bs

```diff
@@ -4709,13 +4709,13 @@ Hue Interpolation</h3>
 <h4 id="hue-longer">
 <dfn export>longer</dfn></h4>
 
-	Angles are adjusted so that |θ₂ - θ₁| ∈ {0, [180, 360)}. In pseudo-Javascript:
+	Angles are adjusted so that |θ₂ - θ₁| ∈ {(-360, -180], [180, 360]}. In pseudo-Javascript:
```

Comment:
```suggestion
	Angles are adjusted so that |θ₂ - θ₁| ∈ {[-360, -180], [180, 360]}. In pseudo-Javascript:
```

Reply:
I don't think it's actually possible to end up with theta2-theta1 == -360, which is why I wrote it the way I did. They can't be in that configuration before the `longer` logic runs, due to the first normalization pass. The first condition is a strict inequality at zero, so if the angles are initially equal, you'll end up in the second case (adding 360 to theta2).